packages/typescript-estree/tests/ast-alignment/parse.ts

```diff
@@ -109,9 +109,18 @@ export function parse(
   } catch (error) {
     const loc = error.loc;
     if (loc) {
-      error.codeFrame = codeFrame(text, loc.line, loc.column + 1, {
-        highlightCode: true,
-      });
+      error.codeFrame = codeFrameColumns(
```

Comment:
`codeFrame` with line and column parameters got deprecated and replaced with `codeFrameColumns`